docs/docs/cloud/aws/scanning.md

```diff
@@ -50,6 +50,13 @@ trivy aws --service s3 --arn arn:aws:s3:::example-bucket
 
 All ARNs with detected issues will be displayed when showing results for their associated service.
 
+## Compliance Spec
```

Comment:
it will be more informative to put more info and the cis checks and link to spec

Reply:
thanks added